ros/lib/app.py

```diff
@@ -31,3 +31,12 @@ def ensure_rbac():
             request=request,
             logger=get_logger(__name__)
         )
+
+
+@app.after_request
+def add_headers(response):
+    response.headers['Content-Security-Policy'] = "script-src 'self';"
+    response.headers['Cache-Control'] = 'no-store'
+    response.headers['Content-Type'] = 'application/json; charset=utf-8;'
```

Comment:
Referring from policies response

Reply:
As per documentation updated the value for charset. Also, changed server name to "API server", I guess it will address your concern in case someone tries to find references using previous naming.